scripts/header-check.sh

```diff
@@ -26,7 +26,7 @@ EXCLUDE_PATTERNS=(
     "core/src/main/resources/*"
     "core/src/test/resources/*"
     "user_tools/src/spark_rapids_pytools/resources/*"
-    "user_tools/docs/resources/*"
```

Comment:
We should exclude `user_tools/docs` from header checks as it contains markdown files